openedx/features/smart_referral/tests/test_tasks.py

```diff
@@ -29,3 +35,46 @@ def task_send_referral_and_toolkit_emails_successfully(self, mock_send_mail):
 
         self.assertEqual(mock_send_mail.call_count, 3)
         mock_send_mail.assert_has_calls(all_rerun_mock_calls)
+
+    @mock.patch('openedx.features.smart_referral.tasks.MandrillClient.send_mail')
+    def test_task_send_referral_follow_up_emails_successfully(self, mock_send_mail):
+        test_email1 = 'test.referral1@example.com'
+        test_email2 = 'test.referral2@example.com'
+
+        SmartReferralFactory(contact_email=test_email1, is_referral_step_complete=False)
+        SmartReferralFactory(contact_email=test_email2, is_referral_step_complete=False)
+
+        contact_emails = [test_email1, test_email2]
+
+        status_first_referral = [
+            {
+                'status': 'sent',
+                'email': test_email1
+            }
+        ]
+
+        status_second_referral = [
+            {
+                'status': 'sent',
+                'email': test_email2
+            }
+        ]
```

Comment:
Adding a blank space after this line would be neater and more consistent.

Reply:
done